Project0/TestCases/test4.in

```diff
@@ -11,7 +11,7 @@ coeff_p1 0 10.0
 eval 420 10
 eval 420 -1
 init 7
-coeff_p1 0 2.23;1 4.7;2 1.81;3 4.04;4 2.73;5 1.04; 6 5.05
-eval 5 99282.23
+coeff_p1 0 2.23;1 4.7;2 1.81;3 4.04;4 2.73;5 1.04;6 5.05
+eval 5 84438.48
```

Comment:
Everything has already been fixed except this, can you just leave this change and remove the other ones? It is causing a merge conflict

Reply:
by 'this' I mean the 84438.48